### ISSUE TITLE
Add DuckDuckGo web search

### ISSUE THREAD

**Issue by rohithpr** [rohithpr/py-web-search#2]

**Comment by rishabhnambiar** [comment]
Can I add DuckDuckGo support now? Is anyone else working on it? @rohithpr

**Comment by knight-shade** [comment]
Is this project still active? Would like to add DuckDuckGo support. @rohithpr

**Comment by rohithpr** [comment]
Nope, I this project isn't maintained anymore and has quite a few bugs even with Google & Bing search. Feel free to fork it though.
